shim/opentelemetry-opencensus-shim/pyproject.toml

```diff
@@ -14,6 +14,7 @@ authors = [
 ]
 classifiers = [
   "Development Status :: 4 - Beta",
+  "Framework :: OpenTelemetry",
   "Intended Audience :: Developers",
```

Comment:
Who decides what classifiers to use? Should we have one for "shim" and "propagator" as well?

Reply:
The package trove_classifiers maintains an official list which is the one used in PyPI. I submitted a request a couple of months ago to include these OpenTelemetry classifiers. I was expecting a bit more of a discussion but the change was merged quite fast.

The classifiers I included were chosen based on the potential number of (mostly third-party) packages that could make use of them.